install_sct

```diff
@@ -29,6 +29,11 @@
 # Authors: PO Quirion, J Cohen-Adad, J Carretero
```

Comment:
One other note: The failing test is due to a platform specific issue with one of GitHub's VMs for CI runners. (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3597)

We haven't figured it out quite yet, but it will cause intermittent errors depending on the machine we get from GitHub. It's luck of the draw, but you can generally just ignore it and hit "re-run jobs" when you see it come up.

Reply:
Thanks for the context! That will definitely save me some time.